### PR TITLE
ci: label script will match key with lang only

### PR DESCRIPTION
**Pull request by Trim21**
```js
mocha.setup('bdd');

const assert = chai.assert;

/**
 * @param {string} patch
 */
const parseChangedLang = (patch) => {
  const set = new Set();
  for (const lang of ['cn', 'de', 'ja', 'fr', 'ko']) {
    const pattern = new RegExp(String.raw`^\+\s*(?:${lang}|'${lang}'):`);
    for (const line of patch.split('\n')) {
      if (pattern.test(line)) {
        // TODO: it'd be nice to add the file/line number here.
        if (!set.has(lang))
          console.log(`label: ${lang} [translation]`);
        set.add(lang);
      }
    }
  }
  return Array.from(set.values());
};

describe('parseChangedLang', () => {
  it('should match only single word', () => {
    assert.deepEqual(parseChangedLang(`+  de: 'hello',`), ['de']);
  });

  it('should match quoted', () => {
    assert.deepEqual(parseChangedLang(`+  'de': 'hello',`), ['de']);
  });

  it('should match only single word', () => {
    assert.deepEqual(parseChangedLang(`+  // parentNode: { classList: () };`), []);
  });

  it('should match only single word', () => {
    assert.deepEqual(parseChangedLang(`+  parentNode: { classList: () };`), []);
  });
});


mocha.run();
```
https://codepen.io/Trim21/pen/gOmeyGo

### CODE REVIEW

**Comment by Trim21** [comment]
should fix the label `de` of https://github.com/quisquous/cactbot/pull/3001

**Comment by valarnin** [comment]
Why use ` ` instead of `\s`? I realize that ` ` should always work due to styling enforcement but syntactically it could be space or tab?

**Comment by Trim21** [comment]
> Why use ` ` instead of `\s`? I realize that ` ` should always work due to styling enforcement but syntactically it could be space or tab?

ok

File: .github/scripts/auto-label.cjs [hunk 1]
```diff
@@ -156,7 +156,7 @@ const getChangedFiles = async (github, identifier) => {
 const parseChangedLang = (patch) => {
   const set = new Set();
   for (const lang of ['cn', 'de', 'ja', 'fr', 'ko']) {
-    const pattern = new RegExp(`^\s*[+].*(?:${lang}|'${lang}'): `);
+    const pattern = new RegExp(String.raw`^\s*[+]\s*(?:${lang}|'${lang}'): `);
```
Comment (quisquous) [inline .github/scripts/auto-label.cjs:159]:
```suggestion
    const pattern = /^[+]\s*(?:${lang}|'${lang}'): /;
```

What about just using `/regex/` as a simplification? Also I think I suggested this, but Id on't think there's any whitespace before the initial `+`.

Comment (Trim21) [reply .github/scripts/auto-label.cjs:159]: not working

Comment (valarnin) [reply .github/scripts/auto-label.cjs]: Maybe it makes sense to add https://eslint.org/docs/rules/prefer-regex-literals ?

Comment (quisquous) [reply .github/scripts/auto-label.cjs]: Oh right sorry, template interpolation.  Sorry, I should have more coffee before making suggestions.  <_<

**Pull request merged**